newhelm/prompt_formatting.py

```diff
@@ -0,0 +1,17 @@
+from newhelm.prompt import ChatPrompt, ChatRole
+
+
+def format_chat(
+    chat: ChatPrompt, *, user_text: str = "user: ", sut_text: str = "assistant: "
```

Comment:
Maybe change the parameters to `user_role: str = "user", sut_role: str = "assistant"` and then do  the colon formatting inside the function`blocks.append(f"{role}: {message.text}")`. That way user's can just pass an unformatted user_role and sut_role without mistakenly ending up with "_usermessage text_". Which I believe is currently the case in `huggingface_client`.

Reply:
My thinking here was to allow callers the flexibility of formatting like `"user - "` and `"USER) "`. But since I managed to mess it up, I think that flexibility is probably not worth it. Updated.